package-lock.json

```diff
@@ -1,14278 +1,8 @@
 {
     "name": "aws-toolkit-vscode",
     "version": "1.47.0-SNAPSHOT",
-    "lockfileVersion": 2,
+    "lockfileVersion": 1,
```

Comment:
This is not correct. Please update your local `npm`.

Reply:
I have updated npm to v18.7.0(latest) and redo npm install, please check if the package-lock.json looks correct